docs/data_management/index.md

```diff
@@ -0,0 +1,189 @@
+# Storage
+
+Research Computing offers several data storage options to meet individual or shared needs of UAB researchers, depending on their requirement and use-cases. The types of storage available, procedures for requesting access, responsibilities, and usage guidelines are detailed in the following sections.
+
+## What Type of Storage Do I Need?
+
+There are multiple locations for data storage both on and off Cheaha each with a specific purpose. You can look at the table below to help determine the storage platform we provide that best matches your needed use-case. If you need additional assistance, please contact [Support](../help/support.md).
+
+{{ read_csv('data_management/res/storage_overview.csv', keep_default_na=False) }}
+
+## What Individual Storage Solutions are Available?
+
+Every Cheaha user has personal directories found at `/home/$USER` (or `$HOME`) and `/data/user/$USER` (or `$USER_DATA`), which are created automatically during account registration. In addition, individual allocations on Long-Term Storage (LTS) are also available upon request. Please read more about [Long-Term Storage](./lts/index.md) and [User Data and Home Directories](./cheaha_storage.md#user-data-and-home-directories).
+
```

Comment:
For the sake of symmetry and completeness, let's add a subsection "How Do I Request Individual Cheaha (GPFS) Storage?". Then we can answer very simply that it is created automatically as part of the account creation process.